ansible/roles/cleaner/scenarios/gelato.yml

```diff
@@ -13,11 +13,21 @@
 # limitations under the License.
 
 ---
-- name: build and install delfin using containerized deployment
-  shell: "{{ item }}"
-  with_items:
-    - docker build -t sodafoundation/delfin .
-    - DELFIN_METRICS_DIR={{ delfin_exporter_prometheus_metrics_dir }} DELFIN_HOST_IP={{ host_ip }} DELFIN_RABBITMQ_USER={{ delfin_rabbitmq_user }} DELFIN_RABBITMQ_PASS={{ delfin_rabbitmq_pass }} docker-compose up -d
-  become: yes
+
```

Comment:
What is being done here? The source file is related to Delfin but the changes are related to multicloud